models/example/order_calculations.sql

```diff
@@ -0,0 +1,30 @@
+with source1 as (
```

Comment:
I would name your CTEs in a way that is helpful for what they are (like orders, customers, joined, aggregated, etc)